todo_app/flask_config.py

```diff
@@ -28,5 +28,10 @@ def __init__(self):
         # Not needed unless doing tests and so we don't need to verify if there is a value or not
         self.LOGIN_DISABLED = os.getenv('LOGIN_DISABLED') == 'True'
 
+        self.LOG_LEVEL = os.getenv('LOG_LEVEL') == 'DEBUG'
```

Comment:
I don't think you mean this; this sets the LOG_LEVEL to either `True` or `False` depending on the environment variable, but we want it to instead be set to a specific log level, e.g. `debug`, `error` etc.
Did you mean to set `DEBUG` as a default?
```suggestion
        self.LOG_LEVEL = os.getenv('LOG_LEVEL', 'DEBUG')
```